code/modules/cm_marines/equipment/sentries.dm

```diff
@@ -1061,39 +1061,32 @@
 
 //Direct replacement to new proc. Everything works.
 /obj/machinery/marine_turret/handle_click(mob/living/carbon/human/user, atom/A, params)
-	if(!operator || !istype(user) || operator != user)
-		return FALSE
-	if(istype(A, /obj/screen))
-		return FALSE
-	if(!manual_override)
+	var/list/modifiers = params2list(params) //Only single clicks.
+	if(modifiers["middle"] || modifiers["shift"] || modifiers["alt"] || modifiers["ctrl"])
```

Comment:
this is really ugly, shouldnt see this outside of code/_onclick/

Reply:
If it weren't for some special mod click interaction on some click relay equipment, I'd have done that at the root already.

Reply:
move the proc into a file in that directory